src/pages/theme/themes/cyan.css

```diff
@@ -0,0 +1,30 @@
+/* Cyan Theme Variables */
+:root {
+  /* Cyan Theme Overrides */
+  --browser-theme-background: #e0f7fa;
+  --browser-theme-text-color: #006064;
+  --browser-theme-primary-highlight: #00acc1;
+  --browser-theme-secondary-highlight: #0097a7;
+  
+  /* Navigation */
+  --peersky-nav-background: #B7FEF5;
+  --peersky-nav-button-color: #004d5f;
+  --peersky-nav-button-hover: #006064;
+  --peersky-nav-button-active: #003d47;
+  --peersky-nav-button-inactive: #4dd0e1;
+  --peersky-background-url-input: #E2FEFB;
+  --peersky-background-find-menu: #b2ebf2;
+
+  /* Settings Page */
+  --settings-bg-primary: #9edfd6;
+  --settings-text-primary: #003d47;
+  --settings-text-secondary: #006064;
```

Comment:
Thanks for addressing changes mentioned here https://github.com/p2plabsxyz/peersky-browser/pull/47/files#r2179623261

For lighter themes like `violet, yellow, green, and cyan`, let's have:
- Lighter colors for settings border.
- Black color on address bar for all light themes `--browser-theme-text-color`.
- I noticed that `--peersky-nav-button-inactive` are too light

> No need to change for dark and light themes, it's perfect.